pkg/proxy/proxier.go

```diff
@@ -791,3 +793,7 @@ func (proxier *Proxier) iptablesHostPortalArgs(destIP net.IP, destPort int, prot
 	args = append(args, "-j", "DNAT", "--to-destination", net.JoinHostPort(proxyIP.String(), strconv.Itoa(proxyPort)))
 	return args
 }
+
+func isClosedError(err error) bool {
+	return strings.Contains(err.Error(), "use of closed network connection")
```

Comment:
Well... I really want to use the error type instead of the string. But this most of the net error is defined privately in go.